### PR TITLE
Changes for running benchmarks on Jenkins

### LINKED ISSUE #688: Capture benchmark results to compare over time
The benchmarks run regularly on Travis, but the results are not captured.
It would be great to keep the results of the runs, and possibly run the benchmarks in an environment where the values are more comparable over time.

### PR DESCRIPTION
**Pull request by 2m**
This PR changes Travis configuration to only compile benchmark source files.

Benchmarks are now being run nightly twice on the Akka Jenkins cluster: https://jenkins.akka.io:8498/job/alpakka-kafka-perf-master/plot/ Last 40 runs are stored and plotted.

Fixes #688

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -17,6 +17,7 @@ val coreDependencies = Seq(
 )
 
 val testkitDependencies = Seq(
+  "com.typesafe.akka" %% "akka-testkit" % akkaVersion,
```
Comment (ennru) [inline build.sbt]: Why did you add it to testkit and not only benchmarks?

Comment (2m) [reply build.sbt]: On the Jenkins node testkit did not compile with missing classes from the testkit dependency. Note that alpakka-kafka testkit already depended transitively on akka-testkit because of akka-stream-testkit. So this only adds transitive dependency explicitly.
